cpp/src/parquet/arrow/arrow_reader_writer_test.cc

```diff
@@ -889,6 +891,24 @@ TYPED_TEST(TestParquetIO, ZeroChunksTable) {
   ASSERT_EQ(1, out->column(0)->num_chunks());
 }
 
+TYPED_TEST(TestParquetIO, TableWithMetadata) {
```

Comment:
`TestParquetIO` is a parameterized test with a lot of different types. This seems overkill here and will further increase compilation times and execution time.

Reply:
Thanks, moved this to TestArrowReaderWriter